containers/R/README.md

```diff
@@ -0,0 +1,59 @@
+# RPython model container 
+This container enables the deployment of R models. It supports both the R and Python runtimes. R functions are called from Python using the RPy2 interface, while the container is built upon the [Python RPC client](https://github.com/ucbrise/clipper/blob/develop/containers/python/rpc.py).
+
+## Prerequisites :
+In addition to the requirements of running clipper, 
+
+1. R must be installed (version:latest , >=3.4)
+2. Python version must be =2.7. 
+3. RPy2 must be installed. For more info, go to <https://rpy2.bitbucket.io/>
+
+## Instructions for Model Deployment and Prediction :
+- Make sure  'r_python_container' image is created from <clipper-root>/RPythonDockerfile and Clipper is running.
+- R models can be formulated and trained in python notebooks using the RPy2 interface, for example :
+
+```py
+import rpy2.robjects as ro
+ro.r('formula=mpg~wt+cyl') #model's formula
+ro.r('dataset=mtcars')  #model's training dataset
+# Create an R model with an RPy2 reference
+model_RPy2 = ro.r('model_R <- lm(formula,data=dataset)') 
+```
+- A previously trained and saved model (in .rds format) can also be loaded as RPy2 object :
+
+```py
+from rpy2.robjects.packages import importr
+base = importr('base')
+self.model = base.readRDS(PATH)  #PATH is the path of saved model.
+```
+
+- Deploy the model :
+
+```py
+Clipper.deploy_R_model(
+   "example_model",1,model_RPy2,"strings"
```

Comment:
Fix this example to match the new signature of `deploy_R_model` (remove the strings argument. Also, please add spaces between the arguments.